library/Director/Objects/GroupMembershipResolver.php

```diff
@@ -392,8 +431,14 @@ protected function addMembershipWhere(ZfSelect $query, $column, & $objects)
         }
 
         $ids = array();
-        foreach ($objects as $object) {
-            $ids[] = (int) $object->get('id');
+        foreach ($objects as $k => $object) {
+            if (is_string($object)) {
+                $ids[] = (int) $k;
+            } elseif (is_numeric($object)) {
```

Comment:
Please change  this to is_int, remove (int) in the line below - then merge to master

Reply:
done